pkg/dns/azure/client/client.go

```diff
@@ -2,7 +2,7 @@ package client
 
 import (
 	"context"
-
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
```

Comment:
Please keep the blank line between the standard-package imports and the next group of imports.